xml/2018-11-04.xml

```diff
@@ -0,0 +1,276 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<email title="Weekly Digest">
+
+  <image
+    width="80"
+    src="https://i.imgur.com/dFBKqHp.jpg"
+    alt="Granola bars on a plate"
+    title="Nature Valley granola bars are great until they start crumbling apart." />
+
+  <hr/>
+
+  <text>
+    Last week was yet again another busy week with Facebook's talk on Search and
+    Machine Learning, NextJump's workshop on React Native and a wholesome Board
+    Games night. In the coming week, we're bringing you even more events, including
+    a Game AI workshop, escape room, sponsor event and more. Keep on reading for
+    more info!
+  </text>
+
+  <hr />
+
+  <text>## General Notices</text>
+
+  <hr />
+
+  <heading>Sponsors' Portal</heading>
+  <text>
+    Searching for internships or placements? Check out our Sponsors Portal, where
+    you can find out more details about our sponsors, read their blog posts and
+    see their open internship/placement positions. You can also upload your CV
+    and directly apply to all of our Gold and Silver tier sponsors!
+
+    You can access the portal at: https://portal.docsoc.co.uk
+  </text>
+
+  <hr />
+
+  <heading>CGCU Mascotry</heading>
+  <image src="https://i.imgur.com/Zm5780r.jpg" />
+  <text>
+    Last Wednesday, seven members of the City and Guilds Union (CGCU), including
+    five from DoCSoc, undertook the incredible feat of stealing the Davy Lamp
+    from the Royal School of Mines Union (RSMU).
+
+    Tensions were high as Memeperial [misleadingly reported](https://www.facebook.com/memeperial/photos/a.1243171632387880/1982226031815766)
+    that it was CGCU which instead succumbed to the hands of the Royal College of
+    Science Union (RSCU) at mascotry. However, with an unexpected twist, the
+    true identities of the mascot's kidnappers [were revealed](https://www.facebook.com/memeperial/photos/a.1243171632387880/1982393761798993).
+
+    The CGCU has yet to issue a ransom to the RSMU. More details can be found
+    on page 20 of [last week's Felix issue](https://f001.backblazeb2.com/file/felixonline/issues/pdfs/felix_1703.pdf).
+  </text>
```

Comment:
<image src="https://i.imgur.com/kVlZUUF.jpg" />


Last Monday, seven members of the City and Guild College Union (CGCU), including five from DoCSoc, undertook the incredible feat of stealing the Davy Lamp from the Royal School of Mines Union (RSMU). Tensions were high as Memeperial [misleadingly reported](https://www.facebook.com/memeperial/photos/a.1243171632387880/1982226031815766) that the Lamp fell into the hands of the Royal College of Science Union (RSCU). However, with an unexpected twist, the true identities of the mascot's kidnappers [were revealed](https://www.facebook.com/memeperial/photos/a.1243171632387880/1982393761798993).

On Thursday the CGCU struck again, raiding the RCSU office to capture their mascot Theta. This time out of the 13 rescuers, a whopping 10 were members of DoCSoc! Great job, and looking forward to the Medics' mascot being stolen!